preferences/debian/requirements/requirements.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: requirements
+spec:
+  # https://www.debian.org/releases/stable/i386/ch03s04.en.html
```

Comment:
Better refer to `https://www.debian.org/releases/stable/amd64/ch03s04.en.html`, that's the release we're aiming for mostly. What about ARM?

Reply:
Done.
Regarding ARM, it has the same requirement as amd64, so IMHO we can keep it as a single preference. See: https://www.debian.org/releases/stable/arm64/ch03s04.en.html
WDYT?

Reply:
Yes, but maybe add it to this comment to.

Reply:
Done.